compose.yaml

```diff
@@ -40,7 +40,18 @@ services:
     build:
       args:
         PHP_VERSION: "8.3"
-        PHP_XDEBUG_VERSION: "3.3.0"
+        PHP_XDEBUG_VERSION: "3.3.2"
+
+  php-8.4:
```

Comment:
feel free to introduce all docker changes as separated PR to merge earlier
(also OK if 8.4 image would run only when someone explicitly ignore version mismatch)